src/utils/getTranslation.ts

```diff
@@ -3,8 +3,8 @@ interface TranslationData {
 }
 
 const locales: { [key: string]: () => Promise<TranslationData> } = {
-  en: () => import("../locales/en.json").then((r) => r.default),
-  pt: () => import("../locales/pt.json").then((r) => r.default)
+  en: () => import("../../locales/en.json").then((r) => r.default),
+  pt: () => import("../../locales/pt.json").then((r) => r.default)
 };
```

Comment:
Would be nice if this logic was in `locales` folder, it makes more sense.

We could organize this way

```
locales/
     - en.json
     - pt.json
     - utils/
           - getTranslation.ts
```  

Because it's a util but it's not a "common util"